tool/gravity/cli/install.go

```diff
@@ -68,6 +68,9 @@ func startInstall(env *localenv.LocalEnvironment, config InstallConfig) error {
 		}
 		return trace.Wrap(err)
 	}
+	if err := config.RunLocalChecks(); err != nil {
```

Comment:
Running local preflight checks outside of the operation plan scope as the two do not mix well.